.rubocop.yml

```diff
@@ -973,3 +973,6 @@ RSpec/NoExpectationExample:
     - ^expect_
     - ^assert_
     - sidekiq_assert_
+
+RSpec/IndexedLet:
+  Enabled: false
```

Comment:
This is a relatively-new rule ([added 4 April 2023](https://github.com/rubocop/rubocop-rspec/pull/1615)) that we clearly don't observe in our existing tests, so I think the best move is to disable the rule.